frontend/src/pages/PipelineDetails.tsx

```diff
@@ -600,12 +603,13 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
 
   public async handleVersionSelectedV2(versionId: string): Promise<void> {
     if (this.state.v2Pipeline) {
-      const selectedVersion = (this.state.v2Versions || []).find(
+      const selectedVersionV1 = (this.state.v1Versions || []).find(v => v.id === versionId);
+      const selectedVersionV2 = (this.state.v2Versions || []).find(
         v => v.pipeline_version_id === versionId,
       );
       const pageTitle = this.state.v2Pipeline.display_name?.concat(
         ' (',
-        selectedVersion?.display_name!,
+        selectedVersionV2?.display_name!,
```

Comment:
/cc @chensun `selectedVersionV2.display_name` is used here in helper function for `PipelineDetailsV2`.